desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -3185,7 +3185,7 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
           var files = $('#inputfile')[0].files[0];
           fd.append('inputfile', files);
           fd.append('fileFormat', ko.mapping.toJSON(viewModel.createWizard.source));
-  
+          ## $.post ??
```

Comment:
Got no luck on how to remove ajax calls.
looking for the $.post method but was not able to send formData.

Reply:
There are already POST guess type and file format calls, can't we piggy back on those?

Reply:
Done, but with help of a new function (upload_local_file in api3.py). @romainr